storagemanager/users/models.py

```diff
@@ -1,10 +1,22 @@
 from django.db import models
-from django.contrib.auth.models import User
+from django.contrib.auth.models import AbstractUser
 from PIL import Image
+from django.conf import settings
+
+
+class User(AbstractUser):
+    username = models.CharField(max_length=255, unique=False)
```

Comment:
Just a nitpick (you don't need to change it if you don't want) - as this is `User` model, don't you think simple `name` would me enough? Also, isn't `unique=False` default so you don't need to set it?

Reply:
Ah, it was already changed in following commit 👍